spec/spec_helper.rb

```diff
@@ -25,17 +25,16 @@ module Shell
 end
 
 # Ruby 1.9 Compat
```

Comment:
we can probably nuke this comment at this point

Reply:
Nuked 💣